### PR TITLE
Add rarity rank and appearanceLikelihood

### PR DESCRIPTION
**Pull request by AlexanderLill**
So we have a basic version in there for now

### CODE REVIEW

File: ionic2/app/pages/poke-detail/poke-detail.page.ts [hunk 1]
```diff
@@ -27,4 +27,18 @@ export class PokeDetailPage {
...
+    let reverseRank = 152 - this.pokemon.rarityRank;
+
+    if (reverseRank == 1) {
+      return "1st";
+    } else if (reverseRank == 2) {
+      return "2nd";
+    } else if (reverseRank == 3) {
+      return "3rd";
+    } else {
+      return reverseRank + "th";
```
Comment (johartl) [inline ionic2/app/pages/poke-detail/poke-detail.page.ts]: That's not gonna work for
21 -> 21st
22 -> 22nd
23 -> 23rd
... and so on. I think this is what you need http://stackoverflow.com/a/13627586

File: ionic2/app/pages/poke-detail/poke-detail.page.html [hunk 2]
```diff
@@ -26,8 +26,14 @@
...
       <ion-card>
-        <ion-card-content>
-        {{pokemon.description}}
+        <ion-card-content text-center>
+          {{pokemon.description}}
+        </ion-card-content>
+      </ion-card>
+      <ion-card>
+        <ion-card-content text-center>
+          {{(pokemon.appearanceLikelihood).toFixed(5)}}% of all Pokémon are a {{pokemon.name}}.<br/>
```
Comment (johartl) [inline ionic2/app/pages/poke-detail/poke-detail.page.html]: What do you think displaying something like
`5 in every 1000 Pokemon appearances are a Bulbasaur.`
`<1 in every 1000 Pokemon appearances are a Mewtu.`
I think it's a lot easier to get an idea of the appearance likelihood compared to showing `0.00455%`.

**Review by johartl — CHANGES_REQUESTED** [review]

File: ionic2/app/pages/poke-detail/poke-detail.page.ts (hunk 1 above)
Comment (AlexanderLill) [reply ionic2/app/pages/poke-detail/poke-detail.page.ts]: thanks for pointing that out :)
fixed

File: ionic2/app/pages/poke-detail/poke-detail.page.html (hunk 2 above)
Comment (AlexanderLill) [reply ionic2/app/pages/poke-detail/poke-detail.page.html]: @johartl Agreed. Done :)

```
<1 in every 1000 Pokémon are a Wartortle.
This puts Wartortle on the 63rd place on the rarity scale.
```

```
About 17 in every 100 Pokémon are a Pidgey.
This puts Pidgey on the 151st place on the rarity scale.
```

```
About 2 in every 1000 Pokémon are a Pikachu.
This puts Pikachu on the 93rd place on the rarity scale.
```

**Pull request merged**